src/components/Pagination/Pagination.js

```diff
@@ -187,23 +203,50 @@ Pagination.propTypes = {
    * Label to be used on button of next
    */
   nextLabel: PropTypes.oneOfType([PropTypes.string, PropTypes.object]),
+  /**
+   * Data that is dispatched when next button is clicked
+   */
+  nextLabelTracking: PropTypes.shape({
+    label: PropTypes.string.isRequired,
+    component: PropTypes.string,
+    customParameters: PropTypes.object
+  }),
   /**
    * Label to be used on button of previous
    */
   previousLabel: PropTypes.oneOfType([PropTypes.string, PropTypes.object]),
+  /**
+   * Data that is dispatched when previous button is clicked
+   */
+  previousLabelTracking: PropTypes.shape({
+    label: PropTypes.string.isRequired,
+    component: PropTypes.string,
+    customParameters: PropTypes.object
+  }),
```

Comment:
The `nextLabelTracking` and `previousLabelTracking` are redundant since they can be covered with the `tracking` prop. The page number is simply the current page number +/- 1.